draft blog for RSE associations & finances blog (#35)
RSE-leaders/researchsoftware.org: _posts/2021-04-02-RSE-financial-organizations.md

---
layout: post
title: "How RSE Associations work financially"
image: /img/financial-org.png
---
Photo by [Micheile Henderson](https://unsplash.com/@micheile) on [Unsplash](https://unsplash.com/).
  
**Authors**

- Daniel S. Katz
- Anne Claire Fouilloux
- Ben van Werkhoven

> - Frank Löffler

Reviewer: ```suggestion
- Frank Löffler
- Stephan Druskat
```

financially function (purely volunteer; income from membership, events,
sponsors, etc.; costs for events, staff, other). While it is now being
posted as a blog, it's really a snapshot in time, as we expect the
RSE associations to change their financial models over time. This post
may be useful to the existing organizations that want to consider
changes, and to new organizations that need to make financial decisions.

* be-RSE: The Belgian Research Software Association is not a formal organisation yet, 
nor is it financially independent.
All efforts are driven by volunteers of various organisations. At the moment, two organisations, 
VIB and KU Leuven are driving the activities.
 
* de-RSE: Formal association with charitable status (includes tax exemptions), financed through membership fees (€60/year for full membership; €30/year for discounted membership), used exclusively to support chartered aims, e.g., local groups, national conference, etc. Financial support for deRSE19 came from corporate and charitable sponsors and the Gesellschaft für Informatik. Society uses academic and commercial infrastructure for work (GWDG's chat and cloud, GitHub).

> * Nordic-RSE: not financial as yet but funded by a “Nordic project”.

Author: ```suggestion
* Nordic-RSE: Does not have its own finances as of yet. All efforts are driven by volunteers for various organizations.
```

Author: @bast - is this ok?

> * Nordic-RSE: not financial as yet but funded by a “Nordic project”.

Author: ```suggestion
* Nordic-RSE: Does not have its own finances as of yet. All efforts are driven by volunteers from various organizations.
```

Reviewer 2: Yes. Thank you!

Author: @bast, is this ok?

---
layout: post
title: "How RSE Associations work financially"
image: /img/financial-org.png
---
Photo by [Micheile Henderson](https://unsplash.com/@micheile) on [Unsplash](https://unsplash.com/).
  
**Authors**

- Daniel S. Katz
- Anne Claire Fouilloux
- Ben van Werkhoven
- Frank Löffler

This blog post is intended to talk about how different RSE associations
financially function (purely volunteer; income from membership, events,
sponsors, etc.; costs for events, staff, other). While it is now being
posted as a blog, it's really a snapshot in time, as we expect the
RSE associations to change their financial models over time. This post
may be useful to the existing organizations that want to consider

> changes, and to new organization that need to make financial decisions.

Reviewer: ```suggestion
changes, and to new organizations that need to make financial decisions.
```

Fix typo.

* be-RSE: The Belgian Research Software Association is not a formal organisation yet, 

> nor is it financially independant.

Reviewer: ```suggestion
nor is it financially independent.
```

Fix typo.

> All efforts is driven by volunteers of various organisations. At the moment, two organisations, 

Reviewer: ```suggestion
All efforts are driven by volunteers of various organisations. At the moment, two organisations, 
```
Fix typo.

VIB and KU Leuven are driving the activities.
 

> * de-RSE: membership fees (~60 €/y): not that much, plan to support
> local groups and national conference

Reviewer: ```suggestion
* de-RSE: financed through membership fees (€60/a for full, €30/a for discounted membership), used exclusively to support chartered aims, e.g., local groups, national conference, etc.
```

Restructure de-RSE section

Author: ```suggestion
* de-RSE: Formal association with charitable status (includes tax exemptions), financed through membership fees (€60/year for full membership; €30/year for discounted membership), used exclusively to support chartered aims, e.g., local groups, national conference, etc. Financial support for deRSE19 came from corporate and charitable sponsors and the Gesellschaft für Informatik. Society uses academic and commercial infrastructure for work (GWDG's chat and cloud, GitHub).
```

> * Nordic-RSE: not financial as yet but funded by a “Nordic project”.

Reviewer: ```suggestion
* Nordic-RSE: no own finances as of yet, funded by a "Nordic project".
```

Have I understood this correctly? Tbc by @bast/@annefou.

Reviewer 2: Sorry for late follow-up. I would change the sentence after the comma to "All efforts are driven by volunteers of various organisations". (wording taken from this text, above)

I think this is more accurate. The financing of the Nordic RSE conference is on the CodeRefinery/NeIC budget but so far has not been used so the efforts have in reality been volunteer efforts and in-kind efforts.
